### PR TITLE
Adding myself to the yearbook 2020 with the changes as requested by the reviewers in #3428

### PR DESCRIPTION
**Pull request by IshanPradhan**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/IshanPradhan/IshanPradhan.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Ishan Pradhan
+institution: Shri Vaishnav Vidyapeeth Vishwavidyalaya
+profile_pic: Ishan.jpg
```
Comment (Mohammed-Shoaib) [inline _data/IshanPradhan/IshanPradhan.md:4]: Incorrect file name, please do one of the following:
1. Change `Ishan.jpg ` to `Ishan.jpeg ` in the markdown file.
2. Rename `Ishan.jpeg ` to `Ishan.jpg`.
I'd suggest you make the change in the markdown file!

Comment (IshanPradhan) [reply _data/IshanPradhan/IshanPradhan.md]: Done! 
Do i need o make another Pull Request?

Comment (Mohammed-Shoaib) [reply _data/IshanPradhan/IshanPradhan.md]: Not needed, your pull request is perfect! :confetti_ball: I approved the changes. :+1:

Comment (IshanPradhan) [reply _data/IshanPradhan/IshanPradhan.md]: Thanks! :D

**Comment by IshanPradhan** [comment]
Hey what should i do with the swag form?
I deleted my last forked repository and created a new one
So should i fill the form again?

**Comment by Mohammed-Shoaib** [comment]
Based on #3366, I don't think you have to, you should be good! :ok_man:

**Pull request merged**